dev-lang/yab/yab-1.7.5.6.recipe

```diff
@@ -4,12 +4,12 @@ language, with special commands designed for Haiku."
 HOMEPAGE="http://yab.orgfree.com/"
 COPYRIGHT="1995-2006 Marc-Oliver Ihm (yabasic)
 	2006-2009 Jan Bungeroth (yab improvements)
-	2013-2017 Jim Saxton (yab improvements)"
+	2013-2018 Jim Saxton (yab improvements)"
 LICENSE="Artistic"
-REVISION="2"
-SOURCE_URI="https://github.com/bbjimmy/YAB/archive/1.7.5.5.tar.gz"
-CHECKSUM_SHA256="ea80d21487c472b7ae11eb4f45bb6dc3468b0e68490a7a1ea2d5c44378dc1853"
-SOURCE_DIR="YAB-1.7.5.5"
+REVISION="1"
+SOURCE_URI="https://github.com/bbjimmy/YAB/archive/1.7.5.6.tar.gz"
+CHECKSUM_SHA256="f579c532547226dec62eb6c639f6e53ee3e82282d62077e864a3bc3889d28c89"
+SOURCE_DIR="YAB-1.7.5.6"
```

Comment:
-SOURCE_DIR="YAB-1.7.5.6"
+SOURCE_DIR="YAB-$portVersion"

Reply:
While at it, could you also add a `SOUCE_FILENAME`?
```diff
-SOURCE_URI="https://github.com/bbjimmy/YAB/archive/1.7.5.6.tar.gz"
+SOURCE_URI="https://github.com/bbjimmy/YAB/archive/$portVersion.tar.gz"
 CHECKSUM_SHA256="f579c532547226dec62eb6c639f6e53ee3e82282d62077e864a3bc3889d28c89"
+SOURCE_FILENAME="YAB-$portVersion.tar.gz"
-SOURCE_DIR="YAB-1.7.5.6"
+SOURCE_DIR="YAB-$portVersion"
```
Thanks!

Useful links:
* [HaikuPorter-Guidelines#archive-name](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#archive-name)
* [HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering)